stwcs/wcsutil/altwcs.py

```diff
@@ -490,6 +455,45 @@ def _getheader(fobj, ext):
     return hdr
 
 
+def _read_alt_wcs(fobj, ext, wcskey=' ', outkey=None, verbose=False):
```

Comment:
I will add docs. `outkey=None` makes the code simply read a WCS (primary or alt) and return its header "as is" without conversion from one key to another. This is essentially "just reading" a WCS.

Reply:
What's wrong with just reading the WCS instead of going through another function with several parameters and complicated logic?

Reply:
I do not understand this comment. One purpose for this function was to consolidate (almost) the same code used in three other places. If you do not like to have the option `outkey=None` I can remove it and the caller will have to set `outkey` to the same value as `wcskey`. This will simplify the logic.